README.md

```diff
@@ -8,7 +8,8 @@ Tekton Dashboard is a general purpose, web-based UI for Tekton Pipelines. It all
 
 ## Pre-requisites
 
-[Tekton Pipelines](https://github.com/tektoncd/pipeline) 0.5.2 or later must be installed in order to use the Tekton Dashboard. Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
+[Tekton Pipelines](https://github.com/tektoncd/pipeline) 0.6.0 must be installed in order to use the Tekton Dashboard release 0.2.0 or master. Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
+To use Tekton Dashboard release 0.1.0 and 0.1.1 you must have [Tekton Pipelines](https://github.com/tektoncd/pipeline) 0.5.1 or 0.5.2 must be installed.
```

Comment:
0.5.1 won't work we have a pull request resource that's only in 0.5.2, let's point people to 0.1.1 and not 0.1.0 (0.1.0 is pointless now really)